### DISCUSSION TITLE
Command line finit.status=off not working?

### DISCUSSION THREAD

**Discussion by markand** [finit-project/finit#364] [Q&A] [answered]
Hello,

I've tried adding `finit.status=off` to the kernel configuration but it seems ineffective as finit status are still shown including banner. I'm running the Git version revision 7cb91854.

Do I miss something?

**Comment by troglobit** [comment, marked as answer]
> I've tried adding `finit.status=off` to the kernel configuration but it seems ineffective as finit status are still shown including banner. I'm running the Git version revision [7cb9185](https://github.com/troglobit/finit/commit/7cb91854b03d892c0838549de16b983a66e60b53).
> 
> Do I miss something?

You need to use the kernel/userspace separator, `-- finit.status.off` works here.

**Comment by markand** [comment]
Hey,

Thanks for the quick reply. I've seen that in the manual page just after submitting the question but you responded so fast I didn't have time to answer myself! I must say I've never seen that syntax (aside when stopping `getopt` in cli programs) into the kernel/init system because systemd can accept `systemd.show_status` even though without `--`. It works now, I still have a unique message at the end though:

```
finit-watchdog[347]: Finit v4.5-rc2 watchdog /dev/watchdog starting ...
```

I have the following line in finit.conf, not sure if it's the culprit:

```
service [12] <pid/syslogd> log watchdog
```

**Reply by troglobit** [reply]
There are many ways to read the kernel command line, many are fragile, e.g. container usercases. The `--` separator has been required for quite a while, it is documented in [doc/cmdline.md](https://github.com/troglobit/finit/blob/master/doc/cmdline.md).

That is the bundled watchdog daemon that comes with Finit.  It is started by default when enabled.  You can disable it with `configure --without-watchdog`.   The message is logged to console if the system log daemon is not yet running.

I do not know what your line in finit.conf refers to, the Finit watchdog daemon is named `watchdogd`.

**Reply by troglobit** [reply]
Alternative to disabling the bundled watchdog daemon: check if dropping `LOG_CONS | ` from the call to `openlog()` in `src/watchdog.c`, line 62, works?  That should do the trick.

**Reply by troglobit** [reply]
OK, I decided to drop that log message, 1ee6c9f. Was only an informational log message and not really necessary.  It's more important that critical log messages are seen, e.g., "Failed connecting to /dev/watchdog"

**Comment by markand** [comment]
It was took from the contrib/alpine/watchdog.conf file:

```
cat contrib/alpine/finit.d/available/watchdog.conf 
service [S123456789] env:-/etc/conf.d/watchdog watchdog $WATCHDOG_OPTS $WATCHDOG_DEV -- System watchdog
```

Thanks for the quick support and for this amazing init system :)

**Reply by troglobit** [reply]
Aha, thanks! I need to update that build script.
